src/components/core/components/CardKeyValue/CardKeyValue.tsx

```diff
@@ -39,14 +39,14 @@ export default function CardKeyValue(props: Props) {
       <TableBody>
         {rows.map((row) => (
           <TableRow key={row.key}>
-            <StyledTableCell hideDivider={hideDivider}>
-              <Typography variant="body1" color="textSecondary" noWrap>
+            <StyledTableCell $hideDivider={hideDivider}>
+              <Typography component='div' variant="body1" color="textSecondary" noWrap>
```

Comment:
Why are you overriding p tag with div?

Reply:
you dont want p's to be in other p's or other div as it will give a dom warning